tests/e2e/util/common/e2e_utils.go

```diff
@@ -90,7 +92,23 @@ func CheckNamespaceEmpty(ctx SpecContext, cl client.Client, ns string) {
 
 func LogDebugInfo() {
 	// General debugging information to help diagnose the failure
-	// TODO: Add more debugging information for others resources
+	// TODO: Add the creation of file with this information to be attached to the test report
+
```

Comment:
`kubectl events -n <namespace>` is also generally useful debug info as well.

Reply:
Thanks. I'll add it